package.json

```diff
@@ -26,6 +26,7 @@
     "tenderly": "yarn tenderly:spawn && yarn env e2e && yarn concurrently --kill-others -n \"rpc-proxy,unchained-proxy\" \"yarn tenderly:rpc-proxy\" \"yarn tenderly:unchained-proxy\"",
     "env": "yarn ts-node --skipProject ./scripts/bootstrap-env.ts",
     "generate:asset-data": "ts-node --compilerOptions '{\"module\":\"commonjs\"}' ./scripts/generateAssetData/generateAssetData.ts",
+    "generate:related-asset-index": "ts-node --compilerOptions '{\"module\":\"commonjs\"}' ./scripts/generateAssetData/generateRelatedAssetIndex.ts",
```

Comment:
So we can run the related asset index generation only, without refetching and regenerating all asset data